templates/service_form.html

```diff
@@ -1,23 +1,30 @@
 {% extends 'base.html' %}
 {% block content %}
-    <form method="POST" enctype="multipart/form-data" id="serviceForm" novalidate>
-        {% csrf_token %}
-        <h2>Introdueix nous serveis:</h2>
-        <br>
-        {{ form.as_p }}
-        <input type="submit" value="Submit" class="btn btn-primary">
-    </form>
-    <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
-    <script>
-        $("#id_category").change(function () {
-            var url = '/load-subcategories';
-            var categoryId = $(this).val();
-            $.ajax({
-                url: url+'/'+categoryId,
-                success: function (data) {
-                    $("#id_subcategory").html(data);
-                }
+    {% if perms.manager.add_additionalservice %}
```

Comment:
I think it's better to check this in the view instead of in the template

Reply:
i believe it is also implemented in the views, so you don't see it in the templated but if you went for it with the url you couldn't also